src/appsignal/opentelemetry.py

```diff
@@ -107,15 +110,16 @@ def start_opentelemetry(config: Config) -> None:
 
 
 def add_instrumentations(
-    config: Config, default_instrumentation_adders=DEFAULT_INSTRUMENTATION_ADDERS
+    config: Config,
+    adders: Mapping[str, DefaultInstrumentationAdder] = DEFAULT_INSTRUMENTATION_ADDERS,
```

Comment:
I simplified annotation because we want to let users specify their own opentelemetry instrumentation, even not explicitly supported by appsignal. At least that's how I interpret the purpose of this argument.

I also renamed it. I removed `default_` because you can pass not only the default adders but custom ones, and removed `instrumentation_` because the function name already says it.

Reply:
I don't think users would call this directly, with custom adders or otherwise. This is for our internal use -- the default instrumentation adders are only passed as a parameter [in order to make it easier to test this function's behavior](https://github.com/appsignal/appsignal-python/blob/1b07fe2b8add104d69f7d6f43f12a2016a702e6f/tests/test_opentelemetry.py#L22-L53). Perhaps the argument should have had an `_` prefix to make that clearer.

If users want to add their own OpenTelemetry instrumentation, they can do so already, without interacting with this function -- the ["instrumentation adders"](https://github.com/appsignal/appsignal-python/blob/1b07fe2b8add104d69f7d6f43f12a2016a702e6f/src/appsignal/opentelemetry.py#L15-L18) don't have an explicit dependency on AppSignal or its OpenTelemetry setup. This is to say, users can already do the following, at the bottom of `__appsignal__.py` or at the top of their application:

```python
from some.arbitrary.package import WhateverInstrumentor
WhateverInstrumentor.instrument()
```

The purpose of these instrumentation adders being named is so that they can be disabled using the `disable_default_instrumentations` config option -- and because the instrumentation's names are a bit tricky, I thought they benefitted from strong typing using literal, so users would get a warning if they typo'ed the config.